maven/tests/org.jboss.tools.maven.configurators.tests/projects/springboot/war/src/test/java/org/jboss/tools/maven/tests/springbootwar/SpringbootWarApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.tools.maven.tests.springbootwar;
```

Comment:
maybe rename the class name Springboot -> SpringBoot for improved naming consistency.
And then we're apparently missing the class headers.